### PR TITLE
tests: fix unit test + add npm command to run unit tests

### PR DESCRIPTION
**Pull request by feloy**

### CODE REVIEW

**Comment by benoitf** [comment]
it says that there are conflicts ?

**Comment by feloy** [comment]
> it says that there are conflicts ?

I rebased

**Review by benoitf — COMMENTED** [review]
running `test:backend` I've:

image

**Comment by benoitf** [comment]
invalid cache on my side, cleaning solved the issue

**Review by benoitf — APPROVED** [review]

**Review by lstocchi — APPROVED** [review]

File: packages/backend/src/studio.spec.ts [hunk 1]
```diff
@@ -55,6 +60,9 @@ afterEach(() => {
 });
 
 test('check activate ', async () => {
+  vi.spyOn(fs.promises, 'readFile').mockImplementation(() => {
+    return "<html></html>";
```
Comment (benoitf) [inline packages/backend/src/studio.spec.ts:64]:
```suggestion
    return '<html></html>';
```

(I suppose linter/formatter won't be happy)

**Comment by slemeur** [comment]
👍

**Pull request merged**